screens/TrackerScreen.js

```diff
@@ -36,215 +44,193 @@ const TrackerScreen = () => {
                   }}
-                >
-                  <Text>{formatDate(date).toString()}</Text>
-                  <Button
-                    onPress={() => setShowDatePicker(true)}
-                    icon={
-                      <Icon name="chevron-down" size={15} color={"black"} />
-                    }
-                    type="clear"
+                />
+                {showDatePicker && (
+                  <DateTimePicker
+                    testID="dateTimePicker"
+                    value={date}
+                    mode={"date"}
+                    is24Hour={true}
+                    display="default"
+                    maximumDate={new Date()}
+                    onChange={onChange}
                   />
-                  {showDatePicker && (
-                    <DateTimePicker
-                      testID="dateTimePicker"
-                      value={date}
-                      mode={"date"}
-                      is24Hour={true}
-                      display="default"
-                      onChange={onChange}
-                    />
-                  )}
-                </View>
-              </Card>
+                )}
+              </View>
+            </Card>
+          </View>
+        </View>
+        <View style={styles.valuecardstyle}>
+          <Card style={styles.valuecard}>
+            <View style={styles.row}>
+              <Text style={styles.label}>Total Spent</Text>
+              <Text style={styles.label}>Total Budget</Text>
+              <Text style={styles.label}>Budget Left</Text>
+            </View>
+            <View style={styles.row}>
+              <Text style={styles.amount}>{`\u20B9${amountUsed}`}</Text>
+              <Text style={styles.amount}>{`\u20B9${budgetAmount}`}</Text>
+              <Text style={styles.amount}>{`\u20B9${budgetLeft}`}</Text>
             </View>
+            <View style={{ flex: 1, width: "100%" }}>
+              <Progress.Bar
+                progress={percentageUsed / 100}
+                width={windowWidth * 0.7}
+                height={20}
+                color={"rgba(128,128,128,0.5)"}
+                borderRadius={10}
+                style={styles.progressBar}
+              ></Progress.Bar>
+              <View style={styles.budgetusedleft}>
+                <Text>{`${percentageUsed}%`}</Text>
+              </View>
+              <View style={styles.budgetusedright}>
+                <Text>{`${100 - percentageUsed}%`}</Text>
+              </View>
+            </View>
+          </Card>
+        </View>
+        <View style={styles.chartContainer}>
+          <View style={styles.chartTitlestyle}>
+            <Text style={styles.chartTitle}>Analysis</Text>
           </View>
-          <View
+          <VictoryPie
+            data={[
+              { x: "Hobbies", y: 350 },
+              { x: "Entertainment", y: 400 },
+              { x: "Travel", y: 550 },
+              { x: "Food", y: 800 },
```

Comment:
Can have one single data source or variable that is parsed to multiple components in the screen. As the idea is to fetch all the data in a single api call, this would be better in future when backend is fulfilled. The data source will have the budget data, chart data as well.